PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/NodeTest.java

```diff
@@ -60,7 +61,9 @@ public void basesTest() {
   
   @Test
   public void genomeTest() {
-    assertEquals("ref2.fasta", node.getGenomes().get(1));
+    Iterator<String> it = node.getGenomes().iterator();
+    it.next();
+    assertEquals("ref2.fasta", it.next());
   }
```

Comment:
Big `Yeay` for API testing. Let's not break tests when I refactor the `model` package, so I don't have to spend hours trying to rewrite them